.github/workflows/run-acceptance-tests.yml

```diff
@@ -525,3 +525,32 @@ jobs:
     name: lint
     if: github.event_name == 'repository_dispatch' ||
       github.event.pull_request.head.repo.full_name == github.repository
+  ensure-examples-gen:
```

Comment:
I don't think we want to add this to the acceptance tests at this time. The convert process is a bit slow, and the examples are currently static. Let's add a Makefile target instead, and then we can update the examples as needed.